modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -5,13 +5,11 @@
 
 :_content-type: PROCEDURE
 [id="oc-mirror-updating-cluster-manifests_{context}"]
-= Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+= Configuring your cluster to use resources generated by oc-mirror
 
-// TODO: Can we find a better title / more generic than calling out the two resources?
+After you have mirrored your image set to the mirror registry, you must apply the generated `ImageContentSourcePolicy`, `CatalogSource`, and release image signature resources into the cluster.
 
-After you have mirrored your image set to the mirror registry, you must install the generated `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
-
-The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry. The `CatalogSource` resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry.
+The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry. The `CatalogSource` resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry. The release image signatures are used to verify the mirrored release images.
```

Comment:
I made very generic additions here and in line 10. Please leave feedback if they could be made more descriptive/accurate.